report/src/main/webapp/app/layout/main/suite/mainView.suite.view.html

```diff
@@ -26,7 +26,18 @@
         <div class="inner">
           <div ng-class="test.getStatus()">
             <div class="test-count test-count-dashboard">
-              <i ng-class="test.getStatus()" class="glyphicon"></i>
```

Comment:
I think icons here are handled by ng-class part - test.getStatus() returns classes that extend some glyphicon classes, so basically we don't need all those ng-if's